src/Collection.js

```diff
@@ -51,14 +63,16 @@ var Collection = React.createClass({
 			var contents = <table className="table table-striped table-hover table-condensed">
 				<thead><tr>
 				{Object.keys(this.state.schema.properties).map(function(name, i) {
-					return <td key={name}>{name}</td>
+					return <td key={name}>{name.replace(/_/g, ' ')}</td>
 				})}
 				</tr></thead>
 				<tbody>
 					{data.map(function(row, i) {
 						return <tr key={i}>
 							{Object.keys(this.state.schema.properties).map(function(name, i) {
-								var text = typeof row[name] == "undefined" || row[name].length < 50 ? row[name] : row[name].substring(0, 45)+'...';
+								var text = row[name] != undefined ? String(row[name]) : ''
+								if( text.length && text[0].match(/^[\[{]/) != null ) text = jsonToHuman(text)
```

Comment:
This seems strange. Why convert row[name] to a string, just to then parse it later? I haven't tested this, but I think it would be more clear than analyzing the text to determine the type. What do you think?
```
If (Array.isArray(row[name])) {
  text = jsonToHuman({"": row[name]})
} else If (typeof row[name] == 'object') 
  text = jsonToHuman(row[name])
else {
  text = String(row[name])
}
```